scala-package/core/src/main/scala/org/apache/mxnet/NDArray.scala

```diff
@@ -408,10 +408,10 @@ object NDArray extends NDArrayBase {
    * @return NDArray of evenly spaced values in the specified range.
    */
   def arange(start: Float, stop: Option[Float] = None, step: Float = 1.0f,
-    repeat: Int = 1, ctx: Context = Context.defaultCtx,
+    repeat: Int = 1, inferRange: Boolean = false, ctx: Context = Context.defaultCtx,
     dType: DType = Base.MX_REAL_TYPE): NDArray = {
-    val params = Map("start" -> start, "step" -> step,
-      "repeat" -> repeat, "ctx" -> ctx.toString, "dtype" -> dType.toString())
+    val params = Map("start" -> start, "step" -> step, "repeat" -> repeat,
+      "infer_range" -> inferRange, "ctx" -> ctx.toString, "dtype" -> dType.toString())
```

Comment:
@lanking520 could you help review this part?

Reply:
Sure. @taliesinb Could you please try to move the newly introduced param to the end of the function in order to bring backward compatibility?

Reply:
please update the doc string for this new parameter. same for symbol

Reply:
I actually prefer you create a new `arange` method with `inferRange` without default in the order you have here, the existing one one call with the default value(false).
Almost all of the methods have context as the last parameter, this one could cause confusion.